publications-2016.bib

```diff
@@ -1486,13 +1486,6 @@ @PhdThesis{Guittet2016
   url       = {https://search.proquest.com/openview/a7b27aaca7f352c4cd8446610eeb79f0/1?pq-origsite=gscholar&cbl=18750&diss=y},
 }
 
-@TechReport{Heltai,
-  author      = {Heltai, L and Rotundo, N},
-  title       = {Error estimates in weighted Sobolev norms for finite element immersed interface methods},
-  institution = {Weierstra{\ss}-Institut f\"{u}r Angewandte Analysis und Stochastik},
-  year        = {2016},
-  url         = {http://www.wias-berlin.de/preprint/2323/wias_preprints_2323.pdf},
-}
```

Comment:
Do you want to add the actual publication to this preprint? See https://doi.org/10.1016/j.camwa.2019.05.029

Reply:
Oh, my bad. It is already part of the 2019 publication list.